gallery.html

```diff
@@ -256,6 +298,49 @@
 <!--[if lte IE 8]>
 <script src="assets/js/ie/respond.min.js"></script><![endif]-->
 <script src="assets/js/main.js"></script>
+<script>
+    $('document').ready(function() {
+        var items = $('.item');
+        var pageCount = Math.ceil(items.length/5);
+        var first = template('|<<', -1);
+        var last = template('>>|',0);
+
+        $(items).hide();
+        for(var i = 0; i < 5; i++){
+            $(items[i]).show();
+        }
+
+        $('#pagination').find('ul').append(first);
```

Comment:
oh we have to build the pagination logic..i thought there might been some ready made code for it already